be/src/exec/schema_scanner/schema_columns_scanner.cpp

```diff
@@ -93,6 +93,41 @@ Status SchemaColumnsScanner::start(RuntimeState *state) {
     return Status::OK();
 }
 
+std::string SchemaColumnsScanner::data_type_to_string(TColumnDesc &desc) {
+    switch (desc.columnType) {
+        case TPrimitiveType::BOOLEAN:
+            return "tinyint";
+        case TPrimitiveType::TINYINT:
+            return "tinyint";
+        case TPrimitiveType::SMALLINT:
+            return "smallint";
+        case TPrimitiveType::INT:
+            return "int";
+        case TPrimitiveType::BIGINT:
+            return "bigint";
+        case TPrimitiveType::LARGEINT:
+            return "bigint unsinged";
+        case TPrimitiveType::FLOAT:
+            return "float";
+        case TPrimitiveType::DOUBLE:
+            return "double";
+        case TPrimitiveType::VARCHAR:
```

Comment:
How to display the length of varchar?

Reply:
the varchar has length display in COLUMN_TYPE, NOT DATA_TYPE